docs/tutorials/particle-filtering.md

```diff
@@ -6,32 +6,27 @@
 
 ## Overview
 
-The purpose of this tutorial is to show how to implement a custom {abbr}`MCL (Monte Carlo Localization)` algorithm using Beluga,
-motivated by the example given by _Probabilistic Robotics [^ProbRob], page 200, figure 8.11_.
+In this tutorial, we will show how to implement a custom {abbr}`MCL (Monte Carlo Localization)` algorithm using Beluga.
 
-This example is about a one-dimensional world, composed by walls and doors. The robot is capable to move in a parallel free and continuous space,
-where it constantly scanning the landmarks of the walls-doors world (the robot also moves only in a one-dimensional world). The last two graphs in the GIF showed above is a graphical representation of this world, where the blue bars represent the walls, the red bars represent the doors and the green bar is the robot.
+This tutorial takes after the one-dimensional particle filter example given in {cite}`thrun2005probabilistic{page 200, figure 8.11}`. There are walls and there are doors in our one-dimensional world, and our one-dimensional robot is free to move along this world. The last two plots in the animation shown above depict this world, where the blue bars are the walls, the red bars are the doors, and the green bar is the robot. While moving, our robot scans for doors and measures its distance to them. There is uncertainty in those measurements, however. There is also uncertainty in our robot proprioception (i.e. its sense of self-movement). We wish to estimate the robot position over time while acknowledging that uncertainty. So we will use a particle filter. The first two plots in the animation show how that position estimate or distribution evolves as the robot moves.
```

Comment:
My English and grammar are not the best, but check if this correction makes sense.
nit: 
```suggestion
This tutorial takes after the one-dimensional particle filter example given in {cite}`thrun2005probabilistic{page 200, figure 8.11}`. There are walls and there are doors in our one-dimensional world, and our one-dimensional robot is free to move along this world. The last two plots in the animation shown above depict this world, where the blue bars are the walls, the red bars are the doors, and the green bar is the robot. While moving, our robot scans for doors and measures its distance to them. There is uncertainty in those measurements, however. There is also uncertainty in our robot's proprioception (i.e. its sense of self-movement). We wish to estimate the robot's position over time while acknowledging that uncertainty. So we will use a particle filter. The first two plots in the animation show how that position estimate or distribution evolves as the robot moves.
```